docs/platforms/unity/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Automatic Instrumentation
+description: "Learn what transactions are captured after tracing is enabled."
+sidebar_order: 20
+---
+
+## Automatically Captured Transactions
+
+The Unity SDK automatically creates transactions for the game's startup and scene loading. Additionally, if opted-in, it generates spans for all `Awake` calls during these operations.
+
+You can manage these settings through the Editor Config Window or <Link to="/platforms/unity/configuration/options/#programmatic-configuration">programmatically</Link>.
+
+![Select automatically generated transactions.](./auto-transaction-config.png)
+
+```csharp
+public override void Configure(SentryUnityOptions options)
+{
+    options.AutoSceneLoadTraces = true;
+    options.AutoStartupTraces = true;
+}
+```
+
+Once automatic instrumentation is configured, the Unity SDK will generate transactions as shown in the image below.
```

Comment:
```suggestion
After you've configured your instrumentation, the Unity SDK will generate traces as shown in the image below.
```
Maybe with the am3 plans, we should start moving away from a transaction-based model.